examples/go-counter/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/marigold-dev/deku/sdks/deku_go_interop"
+)
+
+type message struct {
```

Comment:
What's the point of having a `struct` with only one field? Isn't it ok to use
```go
type message string
```
?

Reply:
Because the message is a json like message, and it has the following shape:
```json
{
"Action": "Increment"
}
```